scripts/check_formatting

```diff
@@ -82,6 +82,19 @@ find . \
     \( -type f -name '*.md' \) \
   \) -exec markdownlint {} +
 
+# We exclude the following URLs from the checks:
+# - https://groups.google.com/g/project-oak-discuss : not publicly accessible
+# - https://crates.io/crates : returns 404
```

Comment:
It would be nice to check crates.io links too.  I guess this is down to https://github.com/rust-lang/crates.io/issues/788, so is there maybe some option to get liche to include `Accept: test/html` ?

Reply:
Created https://github.com/raviqqe/liche/issues/39 to track.